conf/uppmax.config

```diff
@@ -10,31 +10,34 @@ vim: syntax=groovy
  */
 
 process {
-  executor = 'local'
-  cpus = { 1 * task.attempt }
-  memory = { 8.GB * task.attempt }
-  time = { 2.h * task.attempt }
+  executor = 'slurm'
   clusterOptions = { "-A $params.project ${params.clusterOptions ?: ''}" }
   errorStrategy = { task.exitStatus == 143 ? 'retry' : 'finish' }
   maxRetries = 3
   maxErrors = '-1'
 
   $fusioncatcher{
     module = ['bioinfo-tools', 'FusionCatcher/0.99.5a']
-    cpus = { 16 * task.attempt }
-    memory = { 32.GB * task.attempt }
+    time = { 72.h * task.attempt }
```

Comment:
It'd be nice to refactor this to use the base / inheritance config setup that we have in other pipelines. That can be a future PR though..

Reply:
Yeah, you are right. I think I'll do the same with some other comments you made here i.e. leave the in and make an issue so it can be implemented in the future.

Reply:
Sounds good 👍